pom.xml

```diff
@@ -114,6 +115,7 @@ LICENSE file.
     <module>asynchbase</module>
     <module>azuretablestorage</module>
     <module>cassandra</module>
+    <module>cloudspanner</module>
```

Comment:
Would be good to call this googlecloudspanner to match the other Google bindings.